01-js-intro-and-jquery/Topic 1:JavaScriptIntro and JQery part 2/js/script.js

```diff
@@ -0,0 +1,29 @@
+$(document).on('ready', function() {
+    $section = $('.section');
+    $.ajax ({
+        url: 'https://api.spotify.com/v1/search?q=Rolling&type=album',
+        success: function(response) {
```

Comment:
this function can be inside a variable so you can test it, re-use it, also the code would be more clean. Not need to change for this exercise, just be aware for next exercises